api/src/pcapi/admin/custom_views/offer_view.py

```diff
@@ -392,7 +392,7 @@ def _compute_score(view, context, model, name) -> float:
         current_config = request._cached_offer_validation_config
     except AttributeError:
         current_config = offers_repository.get_current_offer_validation_config()
-        request._cached_offer_validation_config = current_config
+        request._cached_offer_validation_config = current_config  # pylint: disable=assigning-non-slot
```

Comment:
Bizarre, je n'ai pas d'avertissement pylint sur cette ligne, que ce soit avec pylint 2.**8**.3 ou 2.**9**.3. Et donc, avec ce commentaire pragma, j'ai au contraire l'avertissement "Useless suppression of 'assigning-non-slot'".

Reply:
D'autre part, est-ce qu'on ne voudrait pas faire des commits distincts pour les 3 modifs (correction du test ; pylint ; mise à jour de Werkzeug) ? C'est plus facile à suivre dans l'historique Git.

Reply:
Je propose : 
- 1 commit BSR pour le fix sur le test
- 1 commit pour la montée de version de werkzeug plus pylint (car les deux sont liés, le commit ne passe pas le check pylint sans les disable.)